<launch>
    <param name="robot_description" command="cat $(find rr_description)/urdf/speedi.urdf"/>

    <node name="speedi_state_publisher" pkg ="robot_state_publisher" type="robot_state_publisher"/>

    <node name="speedi_camera_state_pub" pkg="rr_platform" type="speedi_camera_state_pub" output="screen">
    	<param name="joint_state_uri" value="$(find rr_platform)/launch/camera_joint_state.dat"/>
    	<param name="loaded_joint_state" command="cat $(find rr_platform)/launch/camera_joint_state.dat"/>
        <param name="use_simulator_control" type="bool" value="false" />
    </node>
</launch>
